### PR TITLE
Added version for activesupport

### PR DESCRIPTION
**Pull request by mapreal19**
- Version 5 requires ruby >= 2.2
- Added ruby 2.2 to .travis

### CODE REVIEW

**Comment by ka8725** [comment]
Does it mean the projects used older versions of the gem won't work after this gem update? Do you think is it good?

**Comment by mapreal19** [comment]
We can't use activesupport v5 if we want support for ruby < 2.2

I'm trying to make the suite pass since we have errors on master

screen shot 2016-01-07 at 18 05 41

**Comment by ka8725** [comment]
Yeah, I noticed this. It would be great to fix it. Thanks in advance!

File: .travis.yml [hunk 1]
```diff
@@ -3,3 +3,7 @@ rvm:
   - 1.9.3
   - 2.0.0
   - 2.1.0
+  - 2.2.0
+
+before_install:
```
Comment (mapreal19) [inline .travis.yml]: @ka8725 this fixed the issue on Travis :)

**Comment by mapreal19** [comment]
No problem!

**Comment by ka8725** [comment]
I think that we need to support rails < 4.2 anyway and old bundler versions either. Are there acceptable solutions? I have only 2 ideas now: 1. Do separate branches and maintain them separately. But, actually, I don't find this idea is good enough because it requires manual management and it can lead to bugs and inconsistent state of the branches; 2. Add `ifs` somehow in the gemspec and the travis config, ie if current version of ruby is < 2.2 then allow to install any version of gems and bundler, otherwise install `activesupport` 4.2 or higher.

I think that it's reasonable to try 2nd solution.

**Comment by ka8725** [comment]
In gemfile we could use the `PLATFORMS` feature: http://bundler.io/v1.11/man/gemfile.5.html

**Comment by ka8725** [comment]
Hey, @mapreal19 I've managed to fix this - https://github.com/ka8725/migration_data/pull/14/files

Please, check out master branch in your project, it should work now.

**Comment by ka8725** [comment]
Thank you for your contributing!

**Comment by mapreal19** [comment]
Thanks @ka8725. Awesome work! Just arrived home and seeing this now

**Pull request closed without merging**